src/components/menuChooser.jsx

```diff
@@ -49,6 +50,7 @@ function MenuChooser() {
           img={SizingAppImg}
           title={"Sizing App"}
           ChoosingAppArrow={ChoosingAppArrow}
+          DisplayContent={<div>Lorem ipsum dolor sit amet,<br/>consectetur adipiscing</div>}
```

Comment:
props name should begin with smaller case.

Reply:
addressed